addons/storysource/src/loader/generate-helpers.js

```diff
@@ -0,0 +1,106 @@
+import prettier from 'prettier';
+import { handleADD, handleSTORYOF } from './parse-helpers';
+
+const estraverse = require('estraverse');
+const acorn = require('acorn');
+
+require('acorn-stage3/inject')(acorn);
+require('acorn-jsx/inject')(acorn);
+require('acorn-es7')(acorn);
+
+const acornConfig = {
+  ecmaVersion: '9',
+  sourceType: 'module',
+  ranges: true,
+  locations: true,
+  plugins: {
+    jsx: true,
+    stage3: true,
+    es7: true,
+  },
+};
+
+function generateSourceWithoutUglyComments(source, comments) {
+  let lastIndex = 0;
+  const parts = [source];
+
+  const isUgly = comment => comment.value.trim().indexOf('eslint-') === 0;
```

Comment:
also "global", "flow", "$FlowFixMe", probably others. Let's make it an array of regexp patterns, probably also configurable from outside

Reply:
I've added "global" and "eslint" as defaults. Do you think we need to support flow as well? I'm not familiar with its comments syntax.

Reply:
No, it's good defaults